Aquality.WinAppDriver/src/Aquality.WinAppDriver/Extensions/ShowCommand.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace Aquality.WinAppDriver.Extensions
+{
+    /// <summary>
+    /// Controls how the window is to be shown. 
+    /// This parameter is ignored the first time an application calls ShowWindow, 
+    /// if the program that launched the application provides a STARTUPINFO structure.
+    /// Otherwise, the first time ShowWindow is called, 
+    /// the value should be the value obtained by the WinMain function in its nCmdShow parameter.
+    /// For more information, please visit https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-showwindow.
+    /// </summary>
+    public enum ShowCommand
```

Comment:
I think it will be better to split enum into 2 ones: one for state like Maximized, and one for desired action Minimize

Reply:
this enum is defined by Microsoft in WinAPI (https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-showwindow), I think that splitting would be incorrect, we are using the same function with the same arguments. Also, enum->int value matters here, and now they are ordered by int value